gluten-ut/spark32/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenCastSuite.scala

```diff
@@ -43,69 +43,6 @@ class GlutenCastSuite extends CastSuite with GlutenTestsTrait {
   UDTRegistration.register(classOf[IExampleBaseType].getName, classOf[ExampleBaseTypeUDT].getName)
   UDTRegistration.register(classOf[IExampleSubType].getName, classOf[ExampleSubTypeUDT].getName)
 
-  test("cast array element from integer to string") {
```

Comment:
Could you confirm why these tests were added before? Are they duplicate to the existing Spark tests?

Reply:
Deleted UT and "cast array of integer types to array of double" are duplicates. They are added in #9394 and #9396. They are unrelated to the existing Spark tests.

Reply:
@PHILO-HE Do you have any other comments? I would like to quickly support `cast(array as string)` based on this. Thank you.

Reply:
I am not familiar with the `cast` code of velox. Velox's merger cycle is also relatively long. The conversion performance loss is not large. This is just a temporary quick support idea.